src/pages/home/report/ReportActionItem.js

```diff
@@ -95,8 +126,17 @@ class ReportActionItem extends Component {
                             )}
                             <View style={getReportActionItemStyle(hovered)}>
                                 {!this.props.displayAsGroup
-                                    ? <ReportActionItemSingle action={this.props.action} />
-                                    : <ReportActionItemGrouped action={this.props.action} />}
+                                    ? (
+                                        <ReportActionItemSingle
+                                            action={this.props.action}
+                                            message={message}
+                                        />
+                                    )
+                                    : (
+                                        <ReportActionItemGrouped
+                                            message={message}
```

Comment:
Could we maybe use `children` here instead of `message` (and also in `ReportActionItemSingle`)?